src/System.Buffers/src/System.Buffers.csproj

```diff
@@ -29,12 +30,5 @@
   <ItemGroup Condition="'$(IsPartialFacadeAssembly)' == 'true'">
     <ReferenceFromRuntime Include="System.Private.CoreLib" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)'=='uapaot'">
```

Comment:
I assume you tried compiling passing `/p:TargetGroup=uapaot` to ensure that this works? BTW, I opened #15764 that will add two CI runs for uap and uapaot verticals so that we catch breaks.